src/D2L.Bmx/HtmlXmlHelper.cs

```diff
@@ -0,0 +1,46 @@
+using System.Text.RegularExpressions;
+using System.Xml;
+using D2L.Bmx.Aws;
+
+namespace D2L.Bmx;
+
+internal static partial class HtmlXmlHelper {
+	public static string GetSamlResponseFromLoginPage( string html ) {
+		// TODO: use proper HTML parsing instead of regex
+		var inputXml = new XmlDocument();
+		inputXml.LoadXml( SamlResponseInputRegex().Match( html ).Value );
```

Comment:
I see this comment https://github.com/Brightspace/bmx/pull/322/files#diff-8b1e253817f43a932b584d4f407e885d9f64a848f09317feed09e86d5c3aa1cdL173
but I don't think the HTML Okta returns is malformed. Maybe they cleaned it up at some point?
It's not valid XML, but it is valid HTML that can be parsed more properly using a library like [Html Agility Pack](https://html-agility-pack.net/)

Reply:
Don't recall what that comment was about :/ Think theis current XML parsing is fine.